ci/jobs/analyse.yml

```diff
@@ -34,15 +34,6 @@ analyse/binaries:
   parallel:
     matrix:
       - ARCH: [i386, amd64, armel, armhf, aarch64]
-analyse/correctness:
```

Comment:
Do we have to remove gitlab stuff?

Reply:
These job should be removed from Gitlab - but in `nordvpn-linux-build` project

Reply:
Okay will remove in that project.
On the other hand - shouldn't they be removed here as well? We can't perform these jobs without appropriate Docker containers anyways. And I'm not sure if we should keep all of this stuff if it will not be used anyways?